Sources/ArcGISToolkit/Components/Form/FormInputs/MultiLineTextInput.swift

```diff
@@ -112,3 +113,10 @@ struct MultiLineTextInput: View {
         }
     }
 }
+
+private extension MultiLineTextInput {
+    /// The field type of the text input.
+    var fieldType: FieldType {
+        featureForm!.feature.table!.field(named: element.fieldName)!.type!
+    }
+}
```

Comment:
As just mentioned on Slack, seems it'd be better to have this as a property on `FieldFormElement` rather than having to resolve it at the input level manually with forced optionals. Note that it's also added in `SingleLineTextInput` for now.